docs/examples/dcgan.md

```diff
@@ -67,7 +67,7 @@ by using the [logs](../commands/logs.md) command. If you run the model with defa
 $ floyd logs <JOB_NAME> -t
 ```
 
-Floyd saves any content stored in the `/output` directory at the end of the
+Floyd saves any content stored in the current working directory (`/floyd/home`) at the end of the
```

Comment:
You can drop current - just say "working directory"

Applies to a few other places below